spec/book_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../book'
+require 'yaml'
+
+describe Book do
+  before :each do
+    @book = Book.new('title', 'author')
+  end
+
+  it "should print the book\'s title" do
+    @book.title.should eql 'title'
+  end
+
+  it "should print the book\'s author" do
+    @book.author.should eql 'author'
+  end
+end
```

Comment:
- [ ] All public methods should be tested. Include test for the **add_rental** method in your book class.